java/impl/gcp/src/main/java/co/worklytics/psoxy/SecretManagerConfigService.java

```diff
@@ -110,54 +108,39 @@ public Optional<String> getConfigPropertyAsOptional(ConfigProperty property) {
         }
     }
 
-    private String parameterName(ConfigProperty property) {
-        if (StringUtils.isBlank(this.namespace)) {
-            return property.name();
-        } else {
-            return this.namespace + property.name();
-        }
-    }
-
     @Override
     public boolean acquire(@NonNull String lockId, @NonNull java.time.Duration expires) {
         Preconditions.checkArgument(StringUtils.isNotBlank(lockId), "lockId cannot be blank");
 
-        final SecretName lockSecretName = SecretName.of(projectId, this.namespace + lockId);
-
-        long secretTtl = Math.max(expires.getSeconds(), LOCK_SECRETS_TTL.getSeconds());
+        final SecretName lockSecretName = getLockSecret(lockId);
 
         try (SecretManagerServiceClient client = SecretManagerServiceClient.create()) {
             Secret lockSecret;
             try {
                 lockSecret = client.getSecret(lockSecretName);
             } catch (com.google.api.gax.rpc.NotFoundException e) {
-                //create it
-                Secret initial = Secret.newBuilder()
```

Comment:
That wasn't working due:
1) Lack of perms, as to create a new secret we need to have admin permission on the whole project
2) Creating/dropping a secret is a cost operation for GCP; labels seems its free

Reply:
OK, sure - where's it creating the secret in Terraform? I would except it in `gcp-oauth-secrets` or something, but I don't see it there.